ddtrace/appsec/_api_security/preprocessors.json

```diff
@@ -0,0 +1,108 @@
+{
+  "version": "2.2",
```

Comment:
Are the `version` and `rules` fields required in this file?

Reply:
Yes, no rules deactivate entirely the waf, as, by feature, it will not create a context.

Reply:
isn't this file merged with the rule file anyway? I think it overrides the rules from the builtin file.